_health-care/_js/_components/error-message.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+ * An error message to be included conditionally by question components in an error state.
+ *
+ * @constructor
+ */
+class ErrorMessage extends React.Component {
+  render() {
+    return (
+      <span className="usa-input-error-message" id="input-error-message" role="alert">{this.props.message}</span>
```

Comment:
Actually not sure I can still use shallow render to compare against the golden result because the golden result will have to know the exact id number, which comes from the parent component, but shallow render only renders one level deep.